pinot-common/src/main/java/org/apache/pinot/common/protocols/SegmentCompletionProtocol.java

```diff
@@ -395,6 +396,7 @@ public SegmentStoppedConsuming(Params params) {
     public Response() {
     }
 
+    @JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
This line should be added before line 388

Reply:
good catch, thanks